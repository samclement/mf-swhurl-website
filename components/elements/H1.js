import React from 'react'
import styled from 'styled-components'

const H1 = styled.h1`
  color: black;
`

export default ({ children }) => {
  return <H1>{children}</H1>
}
