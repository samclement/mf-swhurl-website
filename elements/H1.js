import React from 'react'
import styled from 'styled-components'

const H1 = styled.h1`
  color: deeppink;
`
H1.displayName = 'H1'

export default({children}) => {
  return (
    <H1>{children}</H1>
  )
}

