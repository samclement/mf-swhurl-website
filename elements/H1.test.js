/* eslint-env jest */

import React from 'react'
import { render, shallow, mount } from 'enzyme'
import toJson from 'enzyme-to-json'

import H1 from './H1.js'

describe('With Enzyme', () => {
  it('H1 shows "swhurl.com"', () => {
      const wrapper = render(<H1>swhurl.com</H1>)
      expect(wrapper.text()).toEqual('swhurl.com')
    })
})

describe('With Snapshot Testing', () => {
  it('H1 shows "swhurl.com"', () => {
      const wrapper = render(<H1>shwurl.com</H1>).toJSON
      expect(toJson(wrapper)).toMatchSnapshot()
    })
})

