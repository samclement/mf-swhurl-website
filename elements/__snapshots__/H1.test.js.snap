// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing H1 shows "swhurl.com" 1`] = `undefined`;
